ui/src/pages/Create.js

```diff
@@ -299,38 +305,54 @@ class Create extends Component {
     )
   }
 
-  renderPhotoFields() {
-    const photoFilenameSpan = this.state.formPhotoFile
-        ? <span>{this.state.formPhotoFile.name}</span>
-        : null;
-    let photoDeleteButton = null;
-    if (this.state.formPhotoFile) {
-      photoDeleteButton = (
-        <span onClick={(e) => {this.setState({formPhotoFile: ''})}}>X</span>
-      );
-    }
-    return (
-      <PFNotchedOutline label={this.props.intl.formatMessage(MESSAGES.photo)}>
-        {/*
-          * You can't really style an input element, but we don't want to
-          * display it as-is, so we hide the actual input element and use a
-          * styled button as a proxy. The Button below this will call click() on
-          * the input when it's clicked.
-          */}
+  renderPhotoUploadLine() {
+    // You can't really style an input element, but we don't want to display it
+    // as-is, so we hide the actual input element and use a styled button as a
+    // proxy. The Button below this will call click() on the input when it's
+    // clicked.
+    const photoInputField = (
         <input
           className='proxied-upload'
           type='file'
           ref={this.photoUploadInput}
           onChange={(e) => this.setState({formPhotoFile: e.target.files[0]})} />
-        <Button
-          className='pf-button-primary'
-          type='button'
-          onClick={() => this.photoUploadInput.current.click()}
-          disabled={this.state.formPhotoUrl != ""}>
-          {this.props.intl.formatMessage(MESSAGES.uploadPhoto)}
-        </Button>
-        {photoFilenameSpan}
-        {photoDeleteButton}
+    );
+    if (this.state.formPhotoFile) {
+      return (
+          <div className='create-photouploadline'>
+            {photoInputField}
+            <span className='mdc-typography--body1'>
+              {this.state.formPhotoFile.name}
+            </span>
+            <img
+                className='create-photodeletebutton'
+                src='/static/icons/maticon_clear.svg'
+                onClick={(e) => {this.setState({formPhotoFile: ''})}} />
```

Comment:
is the image supposed to be clicked? There might be some accessibility limitations of using onClick on a img tag with no discoverability except styling, maybe adding some aria roles (or wrapping the image in a link?) could help?

Reply:
Oh yeah, good call. I have another PR in progress to improve accessibility, but didn't keep it in mind here.